velox/functions/prestosql/tests/RandTest.cpp

```diff
@@ -74,5 +74,124 @@ TEST_F(RandTest, nonNullInt8) {
   EXPECT_LT(rand(4), 4);
 }
 
+class SecureRandTest : public functions::test::FunctionBaseTest {
```

Comment:
There isn't a need to make a separate SecureRandTest class for a single secureRand function. You can add secureRand function in RandTest itself.